src/libponyrt/actor/actor.c

```diff
@@ -15,6 +15,10 @@
 #include <valgrind/helgrind.h>
 #endif
 
+#define INITIAL_BATCH 100
+#define INCR_BATCH 50
+#define DECR_BATCH 25
```

Comment:
Seems okay to me to leave them as constant for now until somebody requests them to be tunable.